ui-tests/src/test/java/org/openmrs/reference/TransferToWardServiceTest.java

```diff
@@ -45,7 +45,7 @@ public void transferToWardServiceTest() {
         assertTrue(patientVisitsDashboardPage.containsText("Admission"));
 
         patientVisitsDashboardPage.goToTransferToWardServicePage().confirm(ISOLATION_WARD);
-        assertTrue(patientVisitsDashboardPage.containsText("Discharge"));
+        assertTrue(patientVisitsDashboardPage.containsText("Transfer"));
```

Comment:
The `assertion` is amended due to the changes explained [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/525/commits/553894514ff8cea90ab23d36f6da6d7be76ad70a#r685840018)